spec/models/associate_consultant_spec.rb

```diff
@@ -18,6 +18,22 @@
     it { should belong_to(:user) }
   end
 
+  describe '#can_graduate?' do
+    it 'returns true when graduation date has passed' do
+      ac = create(:associate_consultant, :has_reached_graduation_date)
+      ac.can_graduate?.should == true
+    end
+
+    it 'returns false when graduation date has not passed' do
+      ac = create(:associate_consultant)
+      ac.can_graduate?.should == false
+    end
+
+    it 'returns false when an AC has already graduated' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.